geopandas/__init__.py

```diff
@@ -1,27 +1,27 @@
-from geopandas._config import options  # noqa
+from geopandas._config import options  # noqa: F401
```

Comment:
Perhaps instead we could add 
```
[tool.ruff.per-file-ignores]
"geopandas/__init__.py" = ["F401"]
```
to the pyproject.toml, since we'd want to have this noqa on every line ever added to this file?